controllers/flinkcluster/flinkcluster_converter_test.go

```diff
@@ -143,7 +143,7 @@ func getDummyFlinkCluster() *v1beta1.FlinkCluster {
 			APIVersion: "flinkoperator.k8s.io/v1beta1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "flinkjobcluster-sample",
+			Name:      "fjc",
```

Comment:
Name was too big. Some resources like the ingress were not being created as a result.

Reply:
Yeah this should be avoid now with #510; I think it would make sense to have a test to check that all the underlying resources are up.